Description: Use python2 for ntploggps
 There is no python3-gps, so we have to use python (Python 2) for ntploggps.
 We also remove its dependency on the "ntp" module, since we only ship a
 python3-ntp and not a python-ntp.  It only uses the ntp module for printing
 the version number.
Forwarded: not-needed
Origin: vendor
Author: Richard Laager <rlaager@wiktel.com>
Last-Update: 2017-11-05
--- a/ntpclients/ntploggps
+++ b/ntpclients/ntploggps
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/python
 # -*- coding: utf-8 -*-
 
 """\
@@ -30,8 +30,6 @@
 try:
     package = 'GPSD'  # pylint: disable=invalid-name
     import gps
-    package = 'NTP'  # pylint: disable=invalid-name
-    import ntp.util
 except ImportError as e:
     sys.stderr.write("ntploggps: can't find Python %s modules.\n" % package)
     sys.stderr.write("%s\n" % e)
@@ -100,10 +98,6 @@
                     dest='verbose',
                     help="be verbose")
 
-parser.add_argument('-V', '--version',
-                    action="version",
-                    version="ntploggps %s" % ntp.util.stdversion())
-
 args = parser.parse_args()
 
 if args.verbose:
